jenkins/aws/manageSwagger.sh

```diff
@@ -5,5 +5,7 @@ trap 'exit ${RESULT:-1}' EXIT SIGHUP SIGINT SIGTERM
 
 # Note that filename can still overridden via provided parameters
 ${AUTOMATION_DIR}/manageS3Registry.sh -x -y "swagger" -f "swagger.zip" "$@"
+${AUTOMATION_DIR}/manageS3Registry.sh -x -y "swagger" -f "apidoc.zip" "$@"
```

Comment:
-f is overridden in manageImages (second -f on the command line.

think you need to change in there. Also manageBuildReferences for when you promote code.